x/lockup/keeper/msg_server_test.go

```diff
@@ -43,16 +43,16 @@ func (suite *KeeperTestSuite) TestMsgLockTokens() {
 			},
 			expectPass: false,
 		},
-		{
-			name: "try creating multiple coin lockup",
-			param: param{
-				coinsToLock:         sdk.Coins{sdk.NewInt64Coin("stake", 10), sdk.NewInt64Coin("take", 10)}, // setup wallet
-				lockOwner:           sdk.AccAddress([]byte("addr1---------------")),                         // setup wallet
-				duration:            time.Second,
-				coinsInOwnerAddress: sdk.Coins{sdk.NewInt64Coin("stake", 10), sdk.NewInt64Coin("take", 10)},
-			},
-			expectPass: false,
-		},
+		// {
```

Comment:
Personally, I don't feel too comfortable having commented out code, especially test cases. Can we either remove this or un-comment it if it's useful?

Reply:
I had to comment out the test case because i moved the check to `ValidateBasic()` in `msgs.go`, therefore it wasn't being tested when the function `LockTokens(...)` was called

Reply:
So then let's remove it 👍